app/com/linkedin/drelephant/spark/heuristics/SparkApplicationMetricsHeuristic.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.linkedin.drelephant.spark.heuristics
+
+import java.util.Date
+
+import scala.Long
+import scala.collection.JavaConverters
+
+import org.apache.commons.io.FileUtils
+
+import com.linkedin.drelephant.AutoTuner
+import com.linkedin.drelephant.analysis.Heuristic
+import com.linkedin.drelephant.analysis.HeuristicResult
+import com.linkedin.drelephant.analysis.HeuristicResultDetails
+import com.linkedin.drelephant.analysis.Severity
+import com.linkedin.drelephant.configurations.heuristic.HeuristicConfigurationData
+import com.linkedin.drelephant.spark.data.SparkApplicationData
+import com.linkedin.drelephant.spark.fetchers.statusapiv1.StageData
+import com.linkedin.drelephant.spark.fetchers.statusapiv1.StageStatus
+
+/**
+ * A heuristic based on metrics for a Spark app's stages.
+ * This heuristic reports metrics at application level like total input data size, total input records etc.
+ */
+class SparkApplicationMetricsHeuristics(private val heuristicConfigurationData: HeuristicConfigurationData)
+    extends Heuristic[SparkApplicationData] {
+  import SparkApplicationMetricsHeuristics._
+  import JavaConverters._
+
+  override def getHeuristicConfData(): HeuristicConfigurationData = heuristicConfigurationData
+
+  override def apply(data: SparkApplicationData): HeuristicResult = {
+    val evaluator = new Evaluator(this, data)
+
+    val resultDetails = Seq(
+      new HeuristicResultDetails(TOTAL_INPUT_SIZE_IN_MB, "%.4f".format(evaluator.totalInputBytes * 1.0 / (FileUtils.ONE_MB))),
```

Comment:
Nit : Remove paranthesis in (FileUtils.ONE_MB)

Reply:
Done.